README.md

```diff
@@ -14,7 +14,7 @@ Additional description about the project and its features.
 
 To get a local copy up and running follow these simple steps.
 
```

Comment:
Please add `npm install`, `npm start` as these commands are necessary to set the project up locally. Moreover, Please add the `npm test` command so the user is able to run the tests. Kindly make sure the README is professional and has all the information for the end user.

Reply:
Thank you for taking the time to review the project. However, I don't understand what you mean by this requirement. 
Do you want me to add npm start, npm install, and npm test in the readme ? 
Could you be more specific?